diff --git a/src/Plugin/migrate/source/DrupalSqlBase.php b/src/Plugin/migrate/source/DrupalSqlBase.php
index 641ac49..b629369 100644
--- a/src/Plugin/migrate/source/DrupalSqlBase.php
+++ b/src/Plugin/migrate/source/DrupalSqlBase.php
@@ -5,7 +5,7 @@ namespace Drupal\paragraphs\Plugin\migrate\source;
 use Drupal\migrate_drupal\Plugin\migrate\source\DrupalSqlBase as MigrateDrupalSqlBase;
 use Drupal\Component\Plugin\ConfigurablePluginInterface;
 use Drupal\Component\Utility\NestedArray;
-use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\State\StateInterface;
 use Drupal\migrate\Plugin\MigrationInterface;
 
@@ -20,7 +20,7 @@ abstract class DrupalSqlBase extends MigrateDrupalSqlBase implements Configurabl
   /**
    * {@inheritdoc}
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, MigrationInterface $migration, StateInterface $state, EntityManagerInterface $entity_manager) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, MigrationInterface $migration, StateInterface $state, EntityTypeManagerInterface $entity_manager) {
     parent::__construct($configuration, $plugin_id, $plugin_definition, $migration, $state, $entity_manager);
     $this->setConfiguration($configuration);
   }
diff --git a/src/Plugin/migrate/source/d7/FieldableEntity.php b/src/Plugin/migrate/source/d7/FieldableEntity.php
index 8258712..ff3e8a7 100644
--- a/src/Plugin/migrate/source/d7/FieldableEntity.php
+++ b/src/Plugin/migrate/source/d7/FieldableEntity.php
@@ -5,7 +5,7 @@ namespace Drupal\paragraphs\Plugin\migrate\source\d7;
 use Drupal\migrate_drupal\Plugin\migrate\source\d7\FieldableEntity as MigrateFieldableEntity;
 use Drupal\Component\Plugin\ConfigurablePluginInterface;
 use Drupal\Component\Utility\NestedArray;
-use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\State\StateInterface;
 use Drupal\migrate\Plugin\MigrationInterface;
 
@@ -20,7 +20,7 @@ abstract class FieldableEntity extends MigrateFieldableEntity implements Configu
   /**
    * {@inheritdoc}
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, MigrationInterface $migration, StateInterface $state, EntityManagerInterface $entity_manager) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, MigrationInterface $migration, StateInterface $state, EntityTypeManagerInterface $entity_manager) {
     parent::__construct($configuration, $plugin_id, $plugin_definition, $migration, $state, $entity_manager);
     $this->setConfiguration($configuration);
   }
